web-ui/src/main/resources/catalog/js/CatController.js

```diff
@@ -938,6 +938,7 @@
             createPageTpl: "../../catalog/templates/editor/new-metadata-horizontal.html",
             editorIndentType: "",
             allowRemoteRecordLink: true,
+            workflowSearchRecordTypes: ["n", "e"],
```

Comment:
Editing the setting from the UI configuration, converts the array to a string, breaking the search.

You need to edit the value in the UI configuration > JSON Form.

Reply:
Added the UI configuration part 

![image](https://github.com/geonetwork/core-geonetwork/assets/1701393/425eeb40-a316-4b57-b214-ff7d7171d1fe)